geode-docs/tools_modules/geode_apis_compatible_with_redis.html.md.erb

```diff
@@ -68,6 +68,7 @@ The <%=vars.product_name%> APIs compatible with Redis currently supports the fol
 
 **Note**: These commands are supported for Redis 5.
 **Note**: SLOWLOG is implemented as a NoOp for compatibility with some monitoring tools.
+**Note**: Redis accepts Longs as the cursor index for the HSCAN command. The Geode APIs compatible with Redis only accept Integer values. Cursors out of the range of the 32-bit integer type will throw a NumberFormatException.
```

Comment:
I don't think we need to mention the `NumberFormatException`.  The user never sees that because we catch it and return a Redis error response. Maybe we could say something like: 
```
An error containing the message `invalid cursor` is returned if the cursor value is out of range of the 32-bit integer type.
```

By the way, the same is also true for the HSCAN COUNT parameter (Redis accepts up to a 64-bit integer, but we only accept up to a 32-bit integer)... I'm not sure if we want to add that difference as part of this PR as well.